client/src/app/ui/modules/list/components/sort-filter-bar/sort-filter-bar.component.ts

```diff
@@ -244,6 +261,19 @@ export class SortFilterBarComponent<V extends Identifiable> {
         return itemProperty.charAt(0).toUpperCase() + itemProperty.slice(1);
     }
 
+    public get isMobile(): boolean {
+        return this._isMobile;
```

Comment:
You could just pass the subject to the template and use the `async` pipe.

Reply:
Worked on this. I replaced the _isMobile code with the use of observers.